include/avif/avif.h

```diff
@@ -7,7 +7,16 @@
 #include <stddef.h>
 #include <stdint.h>
 
-#ifndef AVIF_API
+// ---------------------------------------------------------------------------
+// Export macros
+
+// AVIF_BUILDING_SHARED_LIBS should only be defined when building shared
+// version of libavif library.
+// AVIF_DLL may only be defined when your project dynamically linking to libavif
+// on Windows, to allow compiler to generate more efficient function call
+// assembly. If you are using libavif as CMake dependency, through CMake
+// package config file or through pkg-config this is defined automatically.
```

Comment:
Yes because in CMakeLists.txt#413 I added it as an interface definition: `target_compile_definitions(avif INTERFACE -DAVIF_DLL)`

If you check the installed file (`${install_prefix}/lib/cmake/libavif/libavif-config.cmake`) you can see something like this:
```cmake
set_target_properties(avif PROPERTIES
  INTERFACE_COMPILE_DEFINITIONS "AVIF_DLL"
  INTERFACE_INCLUDE_DIRECTORIES "${_IMPORT_PREFIX}/include"
)
```